go.mod

```diff
@@ -68,24 +64,26 @@ require (
 	github.com/spf13/cobra v0.0.1
 	github.com/spf13/pflag v1.0.1
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
-	github.com/wercker/stern v0.0.0-20171214125149-b04b5491222d
+	github.com/wercker/stern v0.0.0-20190705090245-4fa46dd6987f
 	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
 	go.etcd.io/bbolt v1.3.3 // indirect
 	go.uber.org/atomic v1.4.0 // indirect
 	go.uber.org/multierr v1.1.0 // indirect
 	go.uber.org/zap v1.10.0 // indirect
-	golang.org/x/net v0.0.0-20190628185345-da137c7871d7
+	golang.org/x/net v0.0.0-20190827160401-ba9fcec4b297
 	google.golang.org/grpc v1.22.0
 	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
 	gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2 // indirect
 	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
 	gopkg.in/go-playground/validator.v8 v8.18.2 // indirect
 	gopkg.in/tchap/go-patricia.v2 v2.3.0 // indirect
-	k8s.io/api v0.0.0-20181213150558-05914d821849
+	k8s.io/api v0.0.0-20190620084959-7cf5895f2711
 	k8s.io/apiextensions-apiserver v0.0.0-20181213153335-0fe22c71c476
-	k8s.io/apimachinery v0.0.0-20181127105237-2b1284ed4c93
-	k8s.io/client-go v0.0.0-20181213151034-8d9ed539ba31
+	k8s.io/apimachinery v0.0.0-20190612205821-1799e75a0719
+	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 	k8s.io/helm v2.12.2+incompatible
 	k8s.io/klog v0.3.2
 	sigs.k8s.io/yaml v1.1.0
 )
+
+replace github.com/wercker/stern => github.com/siggy/stern v0.0.0-20190905052711-472ce9d16dab
```

Comment:
@siggy why not have the fork live under the linkerd org?

Reply:
i opted to put it under my personal account due to concerns around making the fork look like a project that linkerd is officially supporting (for other developers to depend on).

if we as an org are comfortable with those possible implications, i can totally move it, but i was concerned it would send the wrong message.

Reply:
if we're going to have the project depend on forks, i'm slightly more comfortable being owned by the linkerd org. i'm not concerned about branding implications, since there are good tools to highlight the repos we care about

Reply:
sounds good, i'll move it over to the linkerd org.